browser/de_amp/sources.gni

```diff
@@ -0,0 +1,13 @@
+brave_browser_de_amp_sources = [
```

Comment:
why are you using source.gni here instead of BUILD.gn? I don't see anything that would be a circular dependency